src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -90,31 +90,31 @@ def load_command_table(self, _):
         g.custom_command('restore', '_server_restore', supports_no_wait=True)
         g.custom_command('georestore', '_server_georestore', supports_no_wait=True)
         g.command('delete', 'delete', confirmation=True)
-        g.command('show', 'get')
+        g.show_command('show', 'get')
         g.custom_command('list', '_server_list_custom_func')
         g.generic_update_command('update',
                                  getter_name='_server_update_get', getter_type=rdbms_custom,
                                  setter_name='_server_update_set', setter_type=rdbms_custom, setter_arg_name='parameters',
                                  custom_func_name='_server_update_custom_func')
-        g.generic_wait_command('wait', getter_name='_server_mysql_get', getter_type=rdbms_custom)
+        g.custom_wait_command('wait', getter_name='_server_mysql_get')
 
     with self.command_group('postgres server', postgres_servers_sdk, client_factory=cf_postgres_servers) as g:
         g.custom_command('create', '_server_create')
         g.custom_command('restore', '_server_restore', supports_no_wait=True)
         g.custom_command('georestore', '_server_georestore', supports_no_wait=True)
         g.command('delete', 'delete', confirmation=True)
-        g.command('show', 'get')
+        g.show_command('show', 'get')
         g.custom_command('list', '_server_list_custom_func')
         g.generic_update_command('update',
                                  getter_name='_server_update_get', getter_type=rdbms_custom,
                                  setter_name='_server_update_set', setter_type=rdbms_custom, setter_arg_name='parameters',
                                  custom_func_name='_server_update_custom_func')
-        g.generic_wait_command('wait', getter_name='_server_postgresql_get', getter_type=rdbms_custom)
+        g.custom_wait_command('wait', getter_name='_server_postgresql_get')
```

Comment:
You should alter the signature of `custom_wait_command` so that the second positional arg is the getter_name without specifying it.  Then it looks and works more like `custom_show_command` and `custom_command`.

Reply:
`getter_name` IS the second postional argument in the signature, similar to the show commands and generic_update. They all(show/update/wait) also allow the use of the argument as a keyword, so something like `custom_wait_command('wait', '_server_postgresql_get'` is also valid. I've left the usage of the argument as a keyword as the author of the command did.

Reply:
Ah, in that case, I'd just recommend removing it so it looks the rest.